docs/script.js

```diff
@@ -49,8 +70,8 @@ const initializeElements = () => {
 
   expandViewer.getElementsByTagName('model-viewer')[0].classList.add('big')
 
-  viewer.firstChild.appendChild(expandButton)
-  expandViewer.firstChild.appendChild(exitButton)
+  viewerCustom.getElementsByTagName('model-viewer')[0].appendChild(expandButton)
+  expandViewer.getElementsByTagName('model-viewer')[0].appendChild(exitButton)
```

Comment:
Aqui tbm daria pra trocar por `firstChild.firstChild` não sabia se ficava muito ruim

Reply:
Se for muito usado será que vale deixar passar um className como prop?

Reply:
Não entendi, passar um className